### DISCUSSION TITLE
Got the script to work the first time. These errors caused it to crash

### DISCUSSION THREAD

**Discussion by 7zed** [UltimaHoarder/UltimaScraper#1166] [Q&A] [answered]
![1](https://user-images.githubusercontent.com/73954455/117556293-17cfcf80-b01c-11eb-9877-de0d7a054d9b.png)
Fix: onlyfans - file_directory_format is a invalid format since it has no unique identifiers. Use any from this list {media_id},{username}
Fix: metadata_directory_format is a invalid format since it has no unique identifiers. Use any from this list {username}
Fix: starsavn - file_directory_format is a invalid format since it has no unique identifiers. Use any from this list {media_id},{username}
Fix: metadata_directory_format is a invalid format since it has no unique identifiers. Use any from this list {username}

**Comment by M-rcus** [comment, marked as answer]
The placeholder values are supposed to be put as-is, not "renamed". Based on your screenshot I suggest the following:

```
"file_directory_format": "{site_name}/{username}/{api_type}/{media_type}",
"metadata_directory_format": "{site_name}/{username}/Metadata",
```

Keep in mind that you're also editing the `starsavn` config, not the one for `onlyfans` (they are separate). My examples should work for both though.
